src/components/sidebar/sidebar.component.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../icon/icon';
+import Modal from '../modal/modal';
+import { SidebarStyle, SidebarCloseStyle } from './sidebar.style';
+import './sidebar.scss';
```

Comment:
why is scss file neccessary?

Reply:
it necessary because we need to keep animations on the component